ChummerHub/Services/GoogleDrive/DriveHandler.cs

```diff
@@ -109,21 +112,17 @@ private UserCredential AuthorizeGoogleUser()
             }
         }
 
-#pragma warning disable CS0414 // The field 'DriveHandler.flow' is assigned but its value is never used
         private static IAuthorizationCodeFlow flow = null;
-#pragma warning restore CS0414 // The field 'DriveHandler.flow' is assigned but its value is never used
 
 
-#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member 'DriveHandler.DriveHandler(ILogger<Startup>, IConfiguration)'
         public DriveHandler(ILogger<Startup> Logger, IConfiguration configuration)
-#pragma warning restore CS1591 // Missing XML comment for publicly visible type or member 'DriveHandler.DriveHandler(ILogger<Startup>, IConfiguration)'
         {
             Configuration = configuration;
             _logger = Logger;
-            string refreshToken = Configuration["Authentication.Google.RefreshToken"];
+            string refreshToken = Startup.AppSettings["AuthenticationGoogleRefreshToken"];
 
             if (string.IsNullOrEmpty(_folderId))
-                _folderId = Configuration["Authentication.Google.ChummerFolderId"];
+                _folderId = "1VHXJk1IbVj0aRQHzuM3-j_fPsc-hCWO7";//Startup.AppSettings["Authentication.Google.ChummerFolderId"];
```

Comment:
I assume this is a similar situation as in the previous case with Ids being set explicitly instead of from the configuration in docker.

Reply:
yes. It's a pita. FolderId and Refreshtoken are not so senisible, just the Secret is kind of worrieing...